package.json

```diff
@@ -33,7 +33,7 @@
     "generate:proxy:myportal-models": "rimraf generated/myportal && gen-api-models --api-spec api_myportal.yaml --out-dir generated/myportal",
     "generate:proxy:bpd-models": "rimraf generated/bpd && gen-api-models --api-spec api_bpd.yaml --out-dir generated/bpd",
     "generate:proxy:public-models": "rimraf generated/public  && gen-api-models --api-spec api_public.yaml --out-dir generated/public",
-    "generate:api:io": "rimraf generated/io-api && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/io-functions-app/v1.43.0/openapi/index.yaml --no-strict --out-dir generated/io-api --request-types --response-decoders --client",
+    "generate:api:io": "rimraf generated/io-api && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/io-functions-app/master/openapi/index.yaml --no-strict --out-dir generated/io-api --request-types --response-decoders --client",
```

Comment:
I'd rather maintain release version instead of `master ` because in this way we cannot break app build :)

Reply:
Done in https://github.com/pagopa/io-backend/pull/830/commits/15434c00d547ab9ae09e3b48719e133e4cfbcb22